.github/workflows/pr.yaml

```diff
@@ -12,7 +12,8 @@ jobs:
     with:
       docker_push: false
     secrets:
-      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
-      aws_ecr_repository: ${{ secrets.AWS_ECR_REPOSITORY }}
+      aws_region: us-west-2
```

Comment:
why not use variable?

Reply:
https://github.com/elastic-ipfs/event-delivery-lambda/pull/1#discussion_r930239640